File: Daria-Goldenberg/SecondTerm/Task 8/Task 8/Commands/Echo.cs
namespace Task_8.Commands
{
	public class Echo : Command
	{
		public override CommandResult Run(List<string> args)
		{
			foreach (string arg in args)
				Console.Write(arg + " ");
			Console.WriteLine();
			return new CommandResult(new List<string> { }, new List<string> { } );
		}
	}
}

File: ShangareevRustam/Second term/Block 4/2. Simple bash interpreter/app-bash/src/main/kotlin/bashCommands/Echo.kt
package bashCommands

import simpleBashInterpreter.textUtilities.TextUtilities.trimQuotes

object Echo : BashCommand() {

	override fun run(arg: String, input: String, arguments: MutableMap<String, String>): String {
		return trimQuotes(arg.slice(arg.indexOf("echo") + 5 until arg.length))
	}

	override fun isValid(command: String) =
		command.split(" ").any { it != "" } &&
		command.split(" ").first { it != "" } == "echo"

}